settings.gradle

```diff
@@ -24,7 +24,6 @@ include 'fabric-events-lifecycle-v0'
 include 'fabric-item-groups-v0'
 include 'fabric-keybindings-v0'
 include 'fabric-loot-tables-v1'
-include 'fabric-mining-levels-v0'
```

Comment:
Why don't you remove the module directory wholesale if you remove it from gradle build?

Reply:
Oh, shit. It still needs to be included for deprecation purposes.